### DISCUSSION TITLE
建议：在Venus Docs网站站点增加Config相关参数的说明介绍

### DISCUSSION THREAD

**Discussion by AllynSheep** [filecoin-project/venus#4739] [Ideas]
### Checklist

- [X] This is **not** a new feature or an enhancement to the Filecoin protocol. If it is,  please open an [FIP issue](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/venus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the venus forum](https://github.com/filecoin-project/venus/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated improvement to propose.

### Venus component

- [ ] venus daemon - [chain service] chain sync
- [ ] venus auth - [chain service] authentication
- [ ] venus messager - [chain service] message management (mpool)
- [ ] venus gateway - [chain service] gateway
- [ ] venus miner - [chain service] mining and block production
- [X] venus sealer/worker - sealing
- [X] venus sealer - proving (WindowPoSt)
- [X] venus market - storage deal
- [ ] venus market - retrieval deal
- [ ] venus market - data transfer
- [ ] venus light-weight client
- [ ] venus JSON-RPC API
- [ ] Other

### Improvement Suggestion

建议增加
Venussealer、venusmarket目录中config.toml的所有参数做个详细的说明和介绍，便于存储者进行设置和调整。

**Comment by Fatman13** [comment]
收到！doc相关可以在 https://github.com/filecoin-project/venus-docs/issues 里面提。这边以各个组件为主。
